cmd/containerd/command/main_windows.go

```diff
@@ -89,3 +90,12 @@ func setupDumpStacks() {
 		}
 	}()
 }
+
+func initETWLogging() {
```

Comment:
can you just do this in an `init()` instead of calling it in main since we are not handling any of the errors outside of this function?